talent/job-descriptions/applications.md

```diff
@@ -0,0 +1,68 @@
+- src-d/minhashcuda:  fast Weighted MinHash on NVIDIA GPUs. It allows for fast nearly-duplicate repository detection: scan +17 git repositories in 30 minutes to find 1.5M duplicates (the results were published on data.world).  
+- src-d/engine:  a library for running scalable data retrieval pipelines that process any number of Git repositories for source code analysis.  
+- src-d/hercules:  analysis and visualization of a Git repository history. 
+- src-d/gemini: find duplicate code in git repositories.
+
+
+## ROLE
+
+- This position is for a Senior Engineer on the Applications team.
+
+- As an Applications team engineer, you will have a two-fold role:  on one hand it's a Data Engineering, building production-level pipelines to collect and extract data from all git repositories in the world, on another hand it's applied Machine Learning, by productionizing Machine Learning research and wrapping it useful end-user applications.
+
+- You will be expected to have strong backend coding skills in at least one programming language and very good algorithmic skills. Scala coding skills and knowledge about Apache Spark aren't required but will be highly appreciated.
+
+- As you going to be working closely with the Machine Learning team - strong communication skills and interest in Machine Learning, data processing and visualization are more than welcome. 
+
+- You will have to deal with models, built using modern ML stack written in Python and C++ like Tensorflow, Tensorboard, CUDA, so familiarity with any of those tools is a plus.
+
+- Building tools for day to day use by software engineers require good understanding of modern development processes and approaches: Unit/Integration testing, scrum, code reviews, CI/CD, Containers, etc.
+
+
+## CULTURE
+
+- source{d} is a company for developers by developers. We firmly believe in always doing what's best for the community, not necessarily companies. "For developers" is the answer to any strategic question or dilemma in the company. Always For Developers.
+
+- "By developers" is our identity. Everyone in the company is either a developer or has a developer/engineering mindset. We don't hire anyone who doesn't fit into this culture, it's the only way to make sure everyone understands developers and can always defend their interest.
+
+- At the moment, we are 20+ people from 10 different countries working closely together from our office in Madrid, Spain. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely as part of our team does working from Portugal, Estonia, Russia, and others.
+
+- At source{d}, we have a transparent salary policy. Depending on your experience, you can either join us as a Senior Engineer or a Lead. For a Senior Engineer, the salary ranges from €49.000 to €53.000. You can find our net amounts here (in Spanish only).
```

Comment:
here should be a link

Reply:
@campoy The link exists in Lever when you click the word "here". I am adding the whole link in this PR so everyone to know which link we refer to.